### PR TITLE
Hw 11/flask 1

### PR DESCRIPTION
**Pull request by alchemixXx**
very quickly made this task. perhaps there will be a lot of errors :(

### CODE REVIEW

File: flask-lesson_1/homework/fruits/routes.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+from flask import Blueprint, render_template, request
+
+fruits = Blueprint('fruits', __name__)
+
+fruits_list = ['Apple', 'Guava', 'Orange', 'Banana']
+
+
+@fruits.route('/fruits')
+@fruits.route('/fruits/<string:value>', methods=['GET', 'POST', 'DELETE'])
```
Comment (dimigor) [inline flask-lesson_1/homework/fruits/routes.py]: use separate methods for HTTP methods

Comment (alchemixXx) [reply flask-lesson_1/homework/fruits/routes.py]: done

File: flask-lesson_1/homework/fruits/routes.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+#     elif request.method == "DELETE":
+#         fruits_list.remove(value)
+#         return "Fruit was deleted successfully"
+#     else:
+#         return render_template("fruits.html", title='Fruits Page!', fruit_list=fruits_list)
+
+
+
+@fruits.route("/fruits")
+@fruits.route('/fruits/<string:value>', methods=['GET'])
```
Comment (dimigor) [inline flask-lesson_1/homework/fruits/routes.py]: if we type `/fruits` we will see all the fruits, but if we type `/fruits/some/` we have to see info that belongs to only `some` fruit

File: flask-lesson_1/homework/fruits/routes.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+def fruits_get_func():
+    return render_template('fruits.html', title='Fruts Page!', fruits_list=fruits_list)
+
+
+@fruits.route('/fruits/<string:value>', methods=['POST'])
```
Comment (dimigor) [inline flask-lesson_1/homework/fruits/routes.py]: we do not need `value`

File: flask-lesson_1/homework/fruits/routes.py [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+def fruits_post_func(value=None):
+    fruits_list.append(value)
+    return "Fruit was added successfully"
+
+
+@fruits.route('/fruits/<string:value>', methods=['DELETE'])
```
Comment (dimigor) [inline flask-lesson_1/homework/fruits/routes.py]: as above

File: flask-lesson_1/homework/fruits/routes.py [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+def fruits_del_func(value=None):
+    fruits_list.remove(value)
+    return "Fruit was deleted successfully"
```
Comment (dimigor) [inline flask-lesson_1/homework/fruits/routes.py]: add empty line

File: flask-lesson_1/homework/fruits/routes.py (hunk 2 above)
Comment (alchemixXx) [reply flask-lesson_1/homework/fruits/routes.py]: done

File: flask-lesson_1/homework/fruits/routes.py (hunk 3 above)
Comment (alchemixXx) [reply flask-lesson_1/homework/fruits/routes.py]: done

File: flask-lesson_1/homework/fruits/routes.py (hunk 4 above)
Comment (alchemixXx) [reply flask-lesson_1/homework/fruits/routes.py]: done

File: flask-lesson_1/homework/fruits/routes.py (hunk 5 above)
Comment (alchemixXx) [reply flask-lesson_1/homework/fruits/routes.py]: done

**Review by dimigor — APPROVED** [review]

**Pull request merged**